### DISCUSSION TITLE
Make the header nav title text (site title) clickable

### DISCUSSION THREAD

**Discussion by villeodell** [squidfunk/mkdocs-material#3611] [Q&A] [answered]
Just in case someone else comes across #1471 with the same desire -- consider sharing possible solutions here.

**Comment by villeodell** [comment]
One approach if you don't feel like overriding the header in the theme would be to add an onclick or eventListener with [additional javascript](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript) to the header title html element.

mkdocs.yml
```yaml
extra_javascript:
  - javascripts/extra.js

```

javascripts/extra.js
```javascript
document.getElementsByClassName('md-header__title')[0].onclick = function() { location.href = document.location.origin }
```

**Comment by squidfunk** [comment, marked as answer]
Please see https://github.com/squidfunk/mkdocs-material/issues/2044#issuecomment-724072138
